README.md

```diff
@@ -72,7 +72,7 @@ AWS_HANDLER=index.handler
 AWS_MODE=event
 .....
 .....
-AWS_RUNTIME=nodejs
+AWS_RUNTIME=nodejs6.10
```

Comment:
@WebMaitrise Thanks for this. I think this should vary for different users, depending upon their `nodejs` version ?